code/modules/reqs/supplypacks.dm

```diff
@@ -1806,3 +1806,48 @@ FACTORY
 	name = "Howitzer Tanglefoot shell assembly refill"
 	contains = list(/obj/item/factory_refill/howitzer_shell_tfoot_refill)
 	cost = 100
+
+/datum/supply_packs/factory/swat_mask_refill
+	name = "SWAT mask assembly refill"
+	contains = list(/obj/item/factory_refill/swat_mask_refill)
+	cost = 50
+
+/datum/supply_packs/factory/optical_imager_refill
+	name = "Optical imager assembly refill"
+	contains = list(/obj/item/factory_refill/optical_imager_refill)
+	cost = 50
+
+/datum/supply_packs/factory/med_rig_refill
+	name = "M276 pattern medical storage rig assembly refill"
+	contains = list(/obj/item/factory_refill/med_rig_refill)
+	cost = 50
+
+/datum/supply_packs/factory/med_lifesaver_refill
+	name = "M276 pattern pattern lifesaver bag assembly refill"
+	contains = list(/obj/item/factory_refill/med_lifesaver_refill)
+	cost = 50
+
+/datum/supply_packs/factory/module_valk_refill
+	name = "Valkyrie Automedical Armor System assembly refill"
+	contains = list(/obj/item/factory_refill/module_valk_refill)
+	cost = 60
+
+/datum/supply_packs/factory/module_mimir2_refill
+	name = "Mark 2 Mimir Environmental Resistance System assembly refill"
+	contains = list(/obj/item/factory_refill/module_mimir2_refill)
+	cost = 60
+
+/datum/supply_packs/factory/module_tyr2_refill
+	name = "Mark 2 Tyr Armor Reinforcement assembly refill"
+	contains = list(/obj/item/factory_refill/module_tyr2_refill)
+	cost = 60
+
+/datum/supply_packs/factory/module_hlin_refill
+	name = "Hlin Explosive Compensation Module assembly refill"
+	contains = list(/obj/item/factory_refill/module_hlin_refill)
+	cost = 60
+
+/datum/supply_packs/factory/module_surt_refill
+	name = "Hades Incendiary Insulation System assembly refill"
```

Comment:
Called Surt refill in item path and used to make Surt modules yet it's named Hades?